### PR TITLE
feat: add some array utils, under the new Kerosene package

### PR DESCRIPTION
**Pull request by brycehanscomb**
TODO

### CODE REVIEW

File: packages/kerosene/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+    "name": "@kablamo/kerosene",
+    "version": "0.0.1",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/KablamoOSS/kerosene.git",
+        "directory": "packages/kerosene-utils"
```
Comment (ojkelly) [inline packages/kerosene/package.json]: can you update to `/packages/kerosene`

File: packages/kerosene/src/array/contains.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { identity } from "lodash";
```
[the remark this answers is not in the document]
Comment (brycehanscomb) [reply packages/kerosene/src/array/contains.ts]: going to be using lots of lodash all thru kerosene, this is only the first one. all invokations of `noop` or `identity` may as well point to the same instances

File: packages/kerosene/package.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    },
+    "publishConfig": {
+        "access": "public"
+    },
+    "engines": {
+        "node": ">= 6"
+    },
+    "devDependencies": {
+        "@types/jest": "^24.0.0",
+        "@types/react": "^16.8.1",
```
[the remark this answers is not in the document]
Comment (brycehanscomb) [reply packages/kerosene/package.json]: outside the scope of this PR, will do it in another

File: packages/kerosene/src/array/contains.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (ojkelly) [reply packages/kerosene/src/array/contains.ts]: okay cool, if its going to be a dep.

**Pull request merged**